python/main_DDPG.py

```diff
@@ -64,27 +67,29 @@ def test(env,ENV_NAME,num_episodes):
 	agent.test(env, nb_episodes=num_episodes, visualize=False, nb_max_episode_steps=200)
 
 if __name__ == "__main__":
-    ENV_NAME = 'AllVar-v0'
-    env = gym.make(ENV_NAME)
-    parser = argparse.ArgumentParser()
-    parser.add_argument('-rRL','--run_RL', dest = 'run_RL', type = bool,
+	ENV_NAME = 'AllVar-v0'
```

Comment:
This looks like it might have defaulted to 8 spaces? I think you had the spacing correct the first time